cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js

```diff
@@ -274,30 +302,67 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
     Users.deleteViaApi(userData.userId);
     PatronGroups.deleteViaApi(patronGroup.id);
     Conditions.resetConditionViaApi('cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a', 'Maximum outstanding fee/fine balance');
+    Conditions.resetConditionViaApi('72b67965-5b73-4840-bc0b-be8f3f6e047e', 'Maximum number of lost items');
     Location.deleteViaApiIncludingInstitutionCampusLibrary(
       testData.defaultLocation.institutionId,
       testData.defaultLocation.campusId,
       testData.defaultLocation.libraryId,
       testData.defaultLocation.id
     );
   });
+
   it(
     'C350655 Verify automated patron block "Maximum outstanding fee/fine balance" removed after lost item renewed (vega)',
     { tags: [TestTypes.criticalPath, devTeams.vega] },
     () => {
-      cy.visit(SettingsMenu.conditionsPath);
-      Conditions.waitLoading();
-      Conditions.select('Maximum outstanding fee/fine balance');
-      Conditions.setConditionState(blockMessage);
-      cy.visit(SettingsMenu.limitsPath);
-      Limits.selectGroup(patronGroup.name);
-      Limits.setLimit('Maximum outstanding fee/fine balance', '624');
-      // needed for the "Lost Item Fee Policy" so items can get "aged to lost" status
-      cy.wait(230000);
+      const blockMessage = `You have reached maximum outstanding fee/fine balance as set by patron group${getRandomPostfix()}`;
+      setConditionAndLimit(blockMessage, 'Maximum outstanding fee/fine balance', '624');
+      findPatron();
+      UsersCard.waitLoading();
+      Users.checkIsPatronBlocked(blockMessage, 'Borrowing, Renewals, Requests');
+
+      const itemForRenew = itemsData.itemsWithSeparateInstance[Math.floor(Math.random() * 5)];
+      UsersCard.openLoans();
+      UsersCard.showOpenedLoans();
+      UserLoans.openLoan(itemForRenew.barcode);
+      UserLoans.renewItem(itemForRenew.barcode, true);
+      Renewals.renewBlockedPatron(renewComment);
+      RenewConfirmationModal.confirmRenewOverrideItem();
+      OverrideAndRenewModal.confirmOverrideItem();
```

Comment:
Let's check here that this modal exists before the button click

Reply:
done